cases-SEP24/toml.test.js

```diff
@@ -48,10 +48,6 @@ describe("TOML File", () => {
       }
     });
 
-    it("uses TRANSFER_SERVER_SEP0024", async () => {
```

Comment:
why take these out?

Reply:
Because SEP-6 anchors won't have that TOML attribute

Reply:
but this is in cases-sep24

Reply:
Ah crap

Reply:
Ok, added it back, removed the same test in SEP6